src/components/Error.js

```diff
@@ -0,0 +1,17 @@
+import { useRouteError } from "react-router-dom";
+
+const Error = () => {
+
+    const err = useRouteError()
+    console.log(err)
```

Comment:
useRouterError from react-router-dom allows us to handle errors with personalized error page with showcasing data related to corresponding error